examples/analysis/analysis_demo.py

```diff
@@ -5,7 +5,8 @@
 =============
 
 This example simulates some 2AFC data and demonstrates the analysis
-functions ``dprime()`` and ``barplot()``.
+functions :func:`expyfun.analyze.dprime` and
+:func:`expyfun.analyze.barplot()`.
```

Comment:
does it matter that one has parentheses and the other doesn't?  (not trying to bust your chops... just want to get it right myself next time)

Reply:
whoops! apparently not, CircleCI is happy enough:

https://322-11614571-gh.circle-artifacts.com/0/home/ubuntu/expyfun/doc/_build/html/auto_examples/analysis/analysis_demo.html